frontend/public/components/dashboard/dashboards-page/cluster-dashboard/getting-started/explore-admin-features-getting-started-card.tsx

```diff
@@ -40,7 +40,7 @@ export const ExploreAdminFeaturesGettingStartedCard: React.FC = () => {
       id="admin-features"
       icon={<FlagIcon color="var(--pf-global--palette--orange-300)" aria-hidden="true" />}
       title={t('public~Explore new admin features')}
-      titleColor={'var(--pf-global--palette--gold-700)'}
+      titleColor={'var(--co-global--palette--gold-700)'}
```

Comment:
Related to above suggestion.
```suggestion
      titleColor={'var(--co-global--palette--orange-700)'}
```